doc/nrf/libraries/modem/location.rst

```diff
@@ -379,7 +379,10 @@ The following |NCS| applications and samples use this library:
 Limitations
 ***********
 
-* The Location library can only have one application registered at a time. If there is already an application handler registered, another initialization will override the existing handler.
+* The Location library can only have one application registered at a time.
+  If there is already an application handler registered, another initialization will override the existing handler.
+* GNSS API should not be used directly by the application when using the Location library.
```

Comment:
```suggestion
* :ref:`nrfxlib:gnss_interface` should not be used directly by the application when using the Location library.
```

Reply:
I did consider using the link instead, but the title is "Global Satellite Navigation System", not "GNSS". I would rather use "GNSS API" than "Global Satellite Navigation System API". On the other hand, I'm wondering if we should change the title to "GNSS" instead. It seems that it was changed to "Global Navigation Satellite System" by this PR: https://github.com/nrfconnect/sdk-nrfxlib/commit/1e3850de8b28809feb730b805f3033d3c0bcd164

Reply:
I noticed that the name of the API was changed in that PR.
I think we should refer with the link in any case. If the name of the API doesn't make sense, we should change that.

I did have same though that the change done in that PR is not good and I'd use GNSS instead.

Reply:
Ok, I'll update this and make an nrfxlib PR to change the name of the interface.